db.go

```diff
@@ -729,8 +736,8 @@ func FilterList(s *ModelSchema, order string, asc bool, offset int, limit int, a
 }
 
 // Count return the count of records in a table based on a filter
-func Count(a interface{}, query interface{}, args ...interface{}) int {
-	var count int
+func Count(a interface{}, query interface{}, args ...interface{}) int64 {
```

Comment:
Public API change. Can you please keep the signature of the function the same because this will break hundreds of running applications. You can keep internal variable `count` as `int64` then cast it before you return the variable.